test/integration-test/scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/cases/migration/primarykey/TextPrimaryKeyMigrationIT.java

```diff
@@ -92,11 +92,8 @@ public void assertTextPrimaryMigrationSuccess() throws SQLException, Interrupted
         sourceExecuteWithLog(String.format("INSERT INTO %s (order_id,user_id,status) VALUES (%s, %s, '%s')", getSourceTableOrderName(), "1000000000", 1, "afterStop"));
         waitIncrementTaskFinished(String.format("SHOW MIGRATION STATUS '%s'", jobId));
         // TODO The ordering of primary or unique keys for text types is different, but can't reproduce now
-        if (DatabaseTypeUtil.isMySQL(getDatabaseType())) {
-            assertCheckMigrationSuccess(jobId, "DATA_MATCH");
-        } else {
-            assertCheckMigrationSuccess(jobId, "DATA_MATCH");
-        }
+        proxyExecuteWithLog(String.format("CHECK MIGRATION '%s' BY TYPE (NAME='%s')", jobId, "DATA_MATCH"), 2);
```

Comment:
There're several repeated CHECK MIGRATION DistSQL construction, it's better to extract it in super class

Reply:
Because of the addition of case in the unit test, IT will not be modified, rollback it.